digits/model/images/generic/views.py

```diff
@@ -309,8 +309,10 @@ def show(job, related_jobs=None):
             template, context = extension.get_inference_template(form)
             inference_form_html = flask.render_template_string(template, **context)
 
+    generic_form = GenericImageModelForm()
     return flask.render_template(
         'models/images/generic/show.html',
+        form=generic_form,
```

Comment:
I agree. Do you think [this way](https://github.com/NVIDIA/DIGITS/pull/1511/commits/019414fd0ae849b7970dbba2c4f08966ed201efe#diff-46bf113ddd823b115faf1a7edd0675d0) is okay?